ijkmedia/ijkplayer/ff_ffplay.c

```diff
@@ -336,6 +397,8 @@ static int packet_queue_get_or_buffering(FFPlayer *ffp, PacketQueue *q, AVPacket
             new_packet = packet_queue_get(q, pkt, 1, serial);
             if (new_packet < 0)
                 return -1;
+            //设置flag,用于跳出最大缓冲模式
+            ffp->is->seek_cache = 1;
```

Comment:
packet_queue_get_or_buffering() is not called from read_thread.
Is it safe to access it without mutex?